javascript/packages/orchestrator/src/network.ts

```diff
@@ -340,7 +340,10 @@ export class Network {
         `${decorators.cyan("Direct Link")}`,
         `https://polkadot.js.org/apps/?rpc=${wsUri}#/explorer`,
       ],
-      [`${decorators.cyan("Prometheus Link")}`, node.prometheusUri],
+      [
+        decorators.cyan("Log Path"),
+        this.client.tmpDir + "/" + node.name + ".log",
```

Comment:
This will only be valid under `native` client, we should print how to get the logs depending of the provider.

```ts
        switch (networkSpec.settings.provider) {
          case "podman":
            logCommand = `podman logs -f ${podDef.metadata.name}_pod-${podDef.metadata.name}`;
            break;
          case "kubernetes":
            logCommand = `kubectl logs -f ${podDef.metadata.name} -c ${podDef.metadata.name} -n ${namespace}`;
            break;
          case "native":
            logCommand = `tail -f  ${client.tmpDir}/${podDef.metadata.name}.log`;
            break;
        }
```